lib/MiLight/MiLightRadioPL1197_LT8900.cpp

```diff
@@ -0,0 +1,631 @@
+/*
+ * MiLightRadioPL1167_LT89000.cpp
+ *
+ *  Created on: 31 March 2017
+ *      Author: WoodsterDK
+ *
+ *  Very inspired by:
+ *  https://github.com/pmoscetta/authometion-milight/tree/master/Authometion-MiLight
+ *  https://bitbucket.org/robvanderveer/lt8900lib
+ */
+
+#include "MiLightRadioPL1167_LT8900.h"
```

Comment:
I absolutely know what you mean, normally I can't stand different layouts - but here I went for a prof of concept, and again didn't want to mess around too much in you code.
If you like please change according to your preferences.
I'm happy to help with your super project - and feel the code inserted should be something you can accept and up to your standard. (Absolutely no hard feelings : ))